include/deal.II/base/utilities.h

```diff
@@ -178,6 +178,14 @@ namespace Utilities
   unsigned int
   needed_digits(const unsigned int max_number);
 
+  /**
+   * This function allows to cutoff a floating point number @p number
```

Comment:
Not sure if I got the point. Maybe there is a misunderstanding because you think of a fixed floating point notation, e.g., 0.00367. But I am using scientific notation, 3.67e-3, where there is no difference between decimal places before or after the separator. That‘s why I write digits of accuracy. 7 is the third digit of accuracy, not 3. So the „relative error“ caused by this function is bounded by n_digits in my implementation. Cutting off after some decimal places in fixed notation could lead to a relative error that tends to infinity, depending on the order of magnitude of the number. In this sense, the implementation is not that naive.

Reply:
> Maybe there is a misunderstanding because you think of a fixed floating point notation,

Alright, can you please update the documentation of the function to help clear up this misunderstanding? I am sure I won't be the only one with this question. ;-)